index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Portfolio | Edwin Gichuhi</title>
+    </head>
+
+    <body>
+        <nav class="toolbar">
+            <a href="#">John Doe</a>
+            <img src="/images/Icon-Menu.png" alt="menu">
+        </nav>
```

Comment:
:red_circle: **The container of the toolbar section should be a `header` tag. Please wrap your toolbar inside a header tag**
Your code should look like this :point_down: 
```
<header>
      <nav class="toolbar">
            <a href="#">John Doe</a>
            <img src="/images/Icon-Menu.png" alt="menu">
        </nav>
</header>
```

Reply:
Thanks. Fixed.